snippets.go

```diff
@@ -204,3 +205,22 @@ func (s *SnippetsService) SnippetContent(snippet int, options ...OptionFunc) ([]
 
 	return b.Bytes(), resp, err
 }
+
+// Explore gets the list of public snippets.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/snippets.html#explore-all-public-snippets
+func (s *SnippetsService) Explore(opt *ListSnippetsOptions, options ...OptionFunc) ([]*Snippet, *Response, error) {
```

Comment:
For consistency, please make an `ExploreSnippetsOptions` struct instead. I understand that in this case this would be just a copy of the `ListSnippetsOptions` struct, but that way we keep things consistent...

Reply:
Done, I've also renamed `Explore` to `ExploreSnippets` for the same reason.